integration-tests/tests/find-workflow.js

```diff
@@ -30,14 +30,14 @@ const testFindWorkflowFn = _.flow(
      without these lines it will redirect to dev-Terra even if started out at localhost:3000-Terra */
     if (testUrl === 'http://localhost:3000') {
       await findElement(page, clickable({ textContains: 'Yes' }));
-      const yesButtonHrefDetails = (await page.$x('//a[contains(text(), "Yes")]/@href'))[0];
+      const yesButtonHrefDetails = (await page.$$('xpath///a[contains(text(), "Yes")]/@href'))[0];
```

Comment:
Rename of this method, and you have to specify 'xpath/' at the beginning if using an xpath.